src/components/Badge/Badge.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface BadgeProps {
```

Comment:
Replace size number as a predefined set, sm, md, lg,..etc and define sizes to match figma